setup-release-branch.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
`set -eux` here would be nice

Also, run the entire script by Shellcheck - always a good idea.

Reply:
Oh nice, didn't know about Schellcheck!